syntax = "proto3";

package dataTransportation;

service caller{
    rpc sendFile(empty) returns (file_from_server);
    rpc sendOP(operation) returns (operation_status);
}

message operation{
    uint32 op = 1;
    uint32 pos = 2;
    uint32 line = 3;
    string sym = 4;
    uint32 user_id = 5;
}

message operation_status{
    bool flag = 1;
}

message empty{

}

message file_from_server{
    string FILE = 1;
}
